lib/chippy_web/controllers/page_controller.ex

```diff
@@ -21,16 +21,16 @@ defmodule ChippyWeb.PageController do
     |> render("profile.html")
   end
 
-  def profile_save(conn, %{"profile" => %{"user_id" => user_id}}) do
+  def profile_save(conn, %{"profile" => %{"user_id" => user_id} = form}) do
     conn
     |> put_session(:user_id, user_id)
     |> put_flash(:info, "Profile saved successfully.")
-    |> redirect(to: Routes.page_path(conn, :index))
+    |> redirect(to: Map.get(form, "next", Routes.page_path(conn, :index)))
   end
 
   def sprint(conn, %{"sid" => sprint_id}) do
     case get_session(conn, :user_id) do
```

Comment:
Since there are only 2 conditions, it seems like this would be a place for an [`if` statement](https://elixir-lang.org/getting-started/case-cond-and-if.html#if-and-unless), but I couldn't figure out how to do so in a way cleaner than this...

Reply:
I think I remember @nmashton yesterday voicing some argument against `if` which I'm not now remembering...I guess `case` has the handy benefit of capturing the function output for later use?